specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md

```diff
@@ -13,7 +13,6 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: runtime_2_0
```

Comment:
@jhendrixMSFT Please review this change.

Reply:
Why is this removed?

Reply:
This results in a conflict while generating the SDK with autorest. Here's a result of the failed build https://dev.azure.com/azure-sdk/public/_build/results?buildId=51155

We also generate only the runtime_3_0_preview in the other SDKs and only do the batch in Go. So this actually results in an inconsistent SDK release cycle.

Reply:
We support generating multiple API versions in various languages.  Are you saying this API version is no longer supported?
The generation failure appears to be a regression in autorest, I'm following up on that.

Reply:
No, it's still supported. But the default tag for other languages is the `runtime_3_0_preview`. Anyway, if the autorest regression is fixed, it's okay to revert this.

Reply:
@NelsonDaniel this appears to be a regression in autorest, can you please take a look?

Reply:
@NelsonDaniel Hi. Any update about this PR ?

Reply:
Fixed!

Reply:
@MyronFanQiu You can add the tag back

Reply:
I rebased on top of master and added the tag back.